server/client.go

```diff
@@ -64,6 +64,8 @@ type client struct {
 	cmd        string
 	args       [][]byte
 
+	is_authed bool
```

Comment:
The variable format is_authed is not suggested in Go. we could use isAuthed or other camel format instead. :-)